front-end/src/components/player-page/stats/filter-bar.tsx

```diff
@@ -0,0 +1,35 @@
+import {Grid} from "@mui/material";
+import RisenSeasonSelector from "../../selectors/risen-season-selector";
+import RoleSelector from "../../selectors/role-selector";
+import {GameRoles} from "../../../../../Common/Interface/General/gameEnums";
+import React from "react";
+import SeasonModel from "../../../../../Common/models/season.model";
+
+export interface FilterBarProps {
+    seasonConfig?: {
+        seasonId: string,
+        setSeasonId: (seasonId: string) => void,
+        seasons?: SeasonModel[],
+    };
+}
+
+export default function FilterBar(props: FilterBarProps) {
+    return (
+        <Grid spacing={2} container>
+            <Grid item>
+                <RisenSeasonSelector
+                    callBack={() => {}}
+                    sx={{minWidth: '200px', pt: 1, pb: 1}}
```

Comment:
Im curious, does this expand or just stay 200px all the time?

Reply:
200 all the time but this is WIP